addons/account/tests/test_account_move_out_invoice.py

```diff
@@ -3261,7 +3177,7 @@ def test_out_invoice_copy_custom_date(self):
             ]
         })
 
-        copy_invoice = invoice.copy(default={'invoice_date_due': '2018-01-01'})
+        copy_invoice = invoice.copy(default={'invoice_date': '2018-01-01'})
```

Comment:
When you read the docstring
> When creating a refund for a given invoice, the invoice is copied first. This test ensures the payment terms are well recomputed in order to take the new date into account.

The test is not correctly written to start with. It is testing the internals of the refund mecanism instead of testing the refund.
I changed it to
```suggestion
        copy_invoice = invoice.copy(default={
            'invoice_date_due': '2018-01-01',
            'invoice_payment_term_id': False,
        })
```
Which is actually what is done in `Model<account.move.reversal>._prepare_default_reversal`